newswires/client/src/App.tsx

```diff
@@ -99,42 +95,63 @@ export function App() {
 						</EuiHeaderSectionItem>
 					</EuiHeader>
 				)}
-				{status !== 'error' && (
-					<>
-						<EuiShowFor sizes={['xs', 's']}>
-							{view === 'item' ? <Item id={selectedItemId} /> : <Feed />}
-						</EuiShowFor>
-						<EuiShowFor sizes={['m', 'l', 'xl']}>
-							{view === 'item' ? (
-								<EuiResizableContainer className="eui-fullHeight">
-									{(EuiResizablePanel, EuiResizableButton) => (
-										<>
-											<EuiResizablePanel
-												minSize="25%"
-												initialSize={100}
-												className="eui-yScroll"
-											>
-												<Feed />
-											</EuiResizablePanel>
-											<EuiResizableButton />
-											<EuiResizablePanel
-												minSize="30%"
-												initialSize={100}
-												className="eui-yScroll"
-											>
-												<Item id={selectedItemId} />
-											</EuiResizablePanel>
-										</>
-									)}
-								</EuiResizableContainer>
-							) : (
-								<Feed />
-							)}
-						</EuiShowFor>
-					</>
+				{status === 'offline' && (
+					<EuiToast
+						title="You Are Currently Offline"
+						iconType="warning"
+						css={css`
+							border-radius: 0;
+							background: #fdf6d8;
+						`}
+						onClose={() => {}}
```

Comment:
there's a close button on this component, but nothing happens when you click it. I wonder if we should just make it so that you can't dismiss it at all, or else that you can collapse it but not totally dismiss it?

<img width="1009" alt="image" src="https://github.com/user-attachments/assets/6f010a5b-7e1e-4465-9963-f81bbee696db" />